roles/paddles/defaults/main.yml

```diff
@@ -1,9 +1,17 @@
 ---
 paddles_user: paddles
+paddles_db_user: paddles
+paddles_port: 8080
+paddles_statsd_host: ""
+paddles_statsd_prefix: ""
+paddles_sentry_dsn: ""
+
+paddles_containerized: false
+paddles_container_image: quay.io/ceph-infra/paddles:latest
```

Comment:
I guess this image is supposed to be the latest and greatest some how tested before. If so, do we have somewhere a note how can I test paddles image from my local source code before submitting as a PR?

Reply:
Great question. I will write something up to help with that this week.

The latest tag is an alias to master, which is a common convention. If you look [here](https://quay.io/repository/ceph-infra/paddles?tab=tags), you can see some other tags. Each branch of this repo will have containers built automatically. For someone w/o push access to this repo, they'd have to build their own image - and it's possibly to do exactly what I did on quay.io for a forked repo under a personal account for free as well.